hack/generator/pkg/astmodel/testdata/SetArrayOfOptionalFromArrayOfRequired-Direct.golden

```diff
@@ -13,12 +13,14 @@ type Person struct {
 func (person Person) ConvertFrom(source *vHub.Person) error {
 
 	// scores
-	scoreList := make([]*int, len(source.scores))
-	for scoreIndex, score := range source.scores {
-		scoreList[scoreIndex] = &score
+	score1List := make([]*int, len(source.scores))
+	for score1Index, score1Item := range source.scores {
+		score1Item := score1Item
+		score := score1Item
```

Comment:
This is a bit nasty - is there any way we can detect that we've already made a local and just use that? Or is that difficult because of how the code is organised?